### PR TITLE
Disable indexing jobs on Google on non-prod envs

### PR DESCRIPTION
**Pull request by cesidio**

### CODE REVIEW

File: spec/controllers/publishers/vacancies/application_controller_spec.rb [hunk 1]
```diff
@@ -4,23 +4,17 @@
   describe "#update_google_index" do
     let!(:vacancy) { create(:vacancy) }
 
-    context "when in production" do
-      before(:each) do
-        allow(Rails).to receive(:env)
-          .and_return(ActiveSupport::StringInquirer.new("production"))
-      end
+    context "when DisableExpensiveJobs is not enabled" do
+      before(:each) { allow(DisableExpensiveJobs).to receive(:enabled?).and_return(false) }
```
Comment (cpjmcquillan) [inline spec/controllers/publishers/vacancies/application_controller_spec.rb:8]: I'm pretty sure `before` is the same as `before(:each)`

**Review by cpjmcquillan — APPROVED** [review]
I'm happy with this approach - it seems cumbersome to have lots of different flags or a different approach until/unless we need that level of configurability

File: spec/controllers/publishers/vacancies/application_controller_spec.rb (hunk 1 above)
Comment (cesidio) [reply spec/controllers/publishers/vacancies/application_controller_spec.rb:8]: copy and 🍝

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-2929.london.cloudapps.digital>

**Pull request merged**